owncloudApp/src/main/java/com/owncloud/android/ui/fragment/OCFileListFragment.java

```diff
@@ -1004,6 +1029,11 @@ public void listDirectory(OCFile directory/*, boolean onlyOnDevice*/) {
                 directory = storageManager.getFileById(directory.getParentId());
             }
 
+            // If available offline option and folder is not available offline -> list root
+            if(!directory.isAvailableOffline() && isShowingOnlyAvailableOffline()){
+                directory = storageManager.getFileByPath("/");
```

Comment:
Use ROOT_PATH instead of "/" from OCFile.java, as you did in https://github.com/owncloud/android/pull/2468/files#diff-40f7abdfcc80dfb05e4299636743b71aR730